kroxylicious-systemtests/src/main/java/io/kroxylicious/systemtests/installation/kms/aws/AwsKmsClient.java

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright Kroxylicious Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+package io.kroxylicious.systemtests.installation.kms.aws;
+
+import java.net.URI;
+
+import io.kroxylicious.systemtests.Environment;
+
+/**
+ * The interface Aws kms client.
+ */
+public interface AwsKmsClient {
+    /**
+     * Gets aws cmd.
+     *
+     * @return the aws cmd
+     */
+    String getAwsCmd();
+
+    /**
+     * Is available boolean.
+     *
+     * @return the boolean
+     */
+    boolean isAvailable();
+
+    /**
+     * Deploy.
+     */
+    void deploy();
+
+    /**
+     * Gets aws url.
+     *
+     * @return the aws url
+     */
+    URI getAwsUrl();
+
+    /**
+     * Gets region.
+     *
+     * @return the region
+     */
+    String getRegion();
+
+    /**
+     * Gets access key.
+     *
+     * @return the access key
+     */
+    default String getAccessKey() {
+        return Environment.AWS_ACCESS_KEY_ID;
+    }
+
+    /**
+     * Gets secret key.
+     *
+     * @return the secret key
+     */
+    default String getSecretKey() {
+        return Environment.AWS_SECRET_ACCESS_KEY;
+    }
+
+    /**
+     * Delete.
+     */
+    void delete();
```

Comment:
Usually aiming for symmetry in the API is a good approach.  You've gone with `deploy()`, so `undeploy()` is what you'd expect the reciprocal operation to be called.

Reply:
I used `delete` because we have done in the same way for all installation APIs (Strimzi, Vault, AwsKmsClient). I found them synonyms and even more comprehensive using `delete` about what is actually doing.

Honestly I don't see the point to change this everywhere in the framework in this PR.

Reply:
`undeploy` would be quite un-unusual, while I agree that symmetry is usually a smart choice I think using the standard terms is equally valid.

Reply:
ok